### PR TITLE
Put otp behind app's nginx to avoid CORS nonsense

### PR DESCRIPTION
**Pull request by moradology**

### CODE REVIEW

**Comment by flibbertigibbet** [comment]
CORS superseded JSONP in [OpenTripPlanner 0.12](http://opentripplanner.readthedocs.org/en/latest/Changelog/), so it should not be an issue here.  Would be good to modify the commit message.

File: deployment/ansible/roles/nih-wayfinding.app/templates/nginx-nih-wayfinding-app.conf.j2 [hunk 1]
```diff
@@ -1,5 +1,14 @@
 server {
   listen *:80;
   server_name _;
-  root {{ app_home }}/dist;
+
+  location /otp {
+    proxy_pass http://33.33.33.11:8080;
```
Comment (flibbertigibbet) [inline deployment/ansible/roles/nih-wayfinding.app/templates/nginx-nih-wayfinding-app.conf.j2]: Also should set headers here:

```
proxy_set_header X-Real-IP $remote_addr;
proxy_set_header X-Forwarded-Host $host;
proxy_set_header X-Forwarded-Server $host;
proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
```

File: deployment/ansible/roles/nih-wayfinding.app/templates/nginx-nih-wayfinding-app.conf.j2 [hunk 2]
```diff
@@ -1,5 +1,22 @@
...
+  location /otp {
+    proxy_pass http://{{ otp_ip }}:{{ otp_port }};
+    proxy_set_header X-Real-IP $remote_addr;
+    proxy_set_header X-Forwarded-Host $host;
+    proxy_set_header X-Forwarded-Server $host;
+    proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+  }
+
 }
+
```
Comment (sharph) [inline deployment/ansible/roles/nih-wayfinding.app/templates/nginx-nih-wayfinding-app.conf.j2]: Extra newlines.

File: deployment/ansible/group_vars/development [hunk 3]
```diff
@@ -5,3 +5,4 @@ postgresql_password: nih_wayfinding
 postgresql_database: nih_wayfinding
 postgresql_hba_mapping:
   - { type: "host", database: "all", user: "all", address: "33.33.33.1/24", method: "md5" }
+
```
Comment (sharph) [inline deployment/ansible/group_vars/development]: Extra newline. Nothing else in this file has changed.

File: deployment/ansible/roles/nih-wayfinding.app/templates/nginx-nih-wayfinding-app.conf.j2 [hunk 4]
```diff
@@ -1,5 +1,20 @@
 server {
+  root {{ app_home }}/dist;
+  index index.html;
+
   listen *:80;
   server_name _;
-  root {{ app_home }}/dist;
+
+  location / {
+    try_files $uri $uri/ index.html;
```
Comment (sharph) [inline deployment/ansible/roles/nih-wayfinding.app/templates/nginx-nih-wayfinding-app.conf.j2:9]: Why go from `root` to `try_files`?

**Comment by sharph** [comment]
+1

**Pull request merged**